dm/pkg/schema/tracker.go

```diff
@@ -81,7 +104,7 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 	for _, k := range downstreamVars {
 		if _, ok := sessionCfg[k]; !ok {
 			var ignoredColumn interface{}
-			rows, err2 := tidbConn.QuerySQL(tctx, fmt.Sprintf("SHOW VARIABLES LIKE '%s'", k))
+			rows, err2 := downstreamConn.QuerySQL(tctx, fmt.Sprintf("SHOW VARIABLES LIKE '%s'", k))
```

Comment:
I prefer to implement a `QueryRow` function like the sql.DB/Conn does to avoid do a lot of action with `sql.Rows`, but we can do it later.

Reply:
ok, it will be more friendly interface.